docs/development.rst

```diff
@@ -505,6 +505,357 @@ point, with diagrams, and an alternative workflow, see `the git docs
 
 .. _sec_development_demographic_model:
 
+
+********************
+Adding a new species
+********************
+
+---------------------------------------------------
+Which information do I need to have for my species?
+---------------------------------------------------
+
+In `stdpopsim`, we aim to be inclusive and facilitate adding a diverse range of species.
+That said, there are certain basic requirements we have
+for every species added to the :ref:`sec_catalog`.
+We specify these requirements below.
+If you are unsure whether your species satisfies these baseline requirements,
+but you still think it will be useful to add it to `stdpopsim`,
+then we encourage you to open an `issue <http://github.com/popgensims/stdpopsim/issues>`_
+on the GitHub repository to discuss this.
+Others researchers in the community may be able to help you fill in the missing details
+or find other solutions.
+
+Every species added to `stdpopsim` should have the following information available:
+
+1. A chromosome-level genome assembly
+2. Mutation rate (per generation)
+3. Recombination rate (per generation)
+4. A characteristic population size
+5. An average generation time
+
+The **genome assembly** should consist of a list of chromosomes or scaffolds and their lengths.
+Having a good quality assembly with complete chromosomes, or at least very long scaffolds,
+is essential for chromosome-level simulations produced by `stdpopsim`.
+Species with less complete genome builds typically do not have genetic maps
+or good estimates of recombination rates,
+making chromosome-level simulation much less useful.
+Thus, currently, `stdpopsim` only supports adding species with near-complete
+chromosome-level genome assemblies (i.e., close to one contig per chromosome).
+
+An **average mutation rate** and **average recombination rate**
+should be specified for each chromosome (per generation per bp).
+The mutation rate estimate can be based on sequence data from pedigrees, mutation accumulation studies,
+or comparative genomic analysis calibrated by fossil data (i.e., phylogenetic estimates).
+Ideally, one would want to specify a fine-scale chromosome-level **recombination map**,
+since the recombination rate is known to vary widely across chromosomes.
+If a recombination map exists for your species,
+you may specify it separately (see `Adding a genetic map`_).
+Nonetheless, you should specify a default (average) recombination rate for each chromosome.
+If there is no information on the variation of rates among chromosome,
```

Comment:
```suggestion
If there is no information on the variation of rates along the chromosomes,
```

Reply:
I'm referring here to the case where you have an estimate only for the genome-wide average recombination rate and you wish to assign it to all chromosomes. So, variation **along** chromosomes doesn't seem to fit well. Maybe **across**, or **between**?

Reply:
I changed this to **variation of mutation rates across chromosomes** in my latest commit (also split mutation and recombination rates to separate sections here).